pkg/reconciler/artifactbuild/artifactbuild.go

```diff
@@ -496,15 +496,15 @@ func (r *ReconcileArtifactBuild) handleRebuild(ctx context.Context, abr *v1alpha
 }
 
 func CreateABRName(gav string) string {
-	hashedBytes := sha1.Sum([]byte(gav))
+	hashedBytes := sha1.Sum([]byte(gav)) //#nosec
 	hash := hex.EncodeToString(hashedBytes[:])[0:8]
 	namePart := gav[strings.Index(gav, ":")+1:]
 
 	//generate names based on the artifact name + version, and part of a hash
 	//we only use the first 8 characters from the hash to make the name small
 	var newName = strings.Builder{}
 	lastDot := false
-	for _, i := range []rune(namePart) {
+	for _, i := range namePart {
```

Comment:
interesting ... this give you any pause @stuartwdouglas ?  I think it is OK but this name generation is yours